src/raftstore/store/metrics.rs

```diff
@@ -143,9 +143,23 @@ lazy_static! {
             &["cf"]
         ).unwrap();
 
-    pub static ref STORE_KEYS_WRITTEN_COUNTER: Counter =
+    pub static ref STORE_WRITTEN_KEYS_COUNTER: Counter =
```

Comment:
Em, here we may just want to know the written keys rate, if too high, the store may have a high load, so Counter is ok.